challenges.md

```diff
@@ -1,68 +1,110 @@
 # About the Code Challenges
-We have a couple of code challenges designed to give you the opportunity to showcase your skills. We love to see creativity, originality and simplicity of design. All of our projects involve some creativity, research, interest in data, automation and of course amazing instructions. Please make sure to write the most incredible and informative README.md files you can imagine.
+Welcome to the Contrast Code Challenges project page. Code challenges are an important part of the recruiting and hiring process at Contrast. Every engineer/developer, designer or technologist that works at Contrast, full-time or intern submits a code challenge as part of the process to join the team.
 
-There are tons of amazing open and free data sets to play with. Here are a few for inspiration:
+The code challenge is designed to give candidates the ability to showcase their skills in a low stress, extended time model. The code challenge gives you a chance to demonstrate your creativity over a couple of days, rather than in a conference room as part of a technical interview. While we give you ample time, each code challenge is designed to take no more than 4 to 8 hours. We believe that technologists work best in the comfort of their own home, coffee shop or favorite place to code or design. 
 
-* [Data.gov](https://www.data.gov/open-gov/)
-* [NYC Open Data](https://nycopendata.socrata.com/)
-* [Austin, TX Data](https://data.austintexas.gov/browse)
-* [AWS Open Data](https://aws.amazon.com/public-data-sets/)
-* [OpenData Baltimore](https://data.baltimorecity.gov/)
-* [Open Data Sets on GitHub](http://www.kdnuggets.com/2015/04/awesome-public-datasets-github.html)
-* [Firebase Data Sets](https://www.firebase.com/docs/open-data/)
+Most importantly we do the challenge for you. Anyone submitting a code challenge, owns their own code, whether they put it on GitHub or another site. Our theory is that you are putting time into the interview process. We want you to get something out of the effort. What better than something to add to your portfolio?
 
-# Project #1: For the Front-End Enthusiast and JavaScript Aficionados
-We would like for you to pick a single data set or multiple temporal data sets that can be combined into a shareable data set for visualization purposes. Finding a perfect data set is not as important as how it is used.
+One last important item...please do not reference Contrast on your project as we do not want to introduce the possibility of someone plagiarizing your work. We rotate projects about 1-2 times a year.
```

Comment:
One last important item...please do not reference Contrast on your project as we do not want to introduce the possibility of someone plagiarizing your work. We rotate projects several times a year.